index.html

```diff
@@ -27,12 +27,7 @@
     </header>
     <section class="main">
         <div class="main__left-menu">
-            <select class="main__left-select">
-                <option value="now_playing">Now playing</option>
-                <option value="popular">Popular</option>
-                <option value="top_rated">Top rated</option>
-                <option value="upcoming">Upcoming</option>
-            </select>
+            <select class="main__left-select"></select>
```

Comment:
Add a new class to this element if you are going to manipulate with jQuery. For this element add this class 'js-left-menu'.